src/index.jsx

```diff
@@ -67,6 +67,12 @@ import ErrorWorkspacePlaceholder from './Workspace/ErrorWorkspacePlaceholder';
 import { ReduxStudyViewer, ReduxSingleStudyViewer } from './StudyViewer/reduxer';
 import NotFound from './components/NotFound';
 import ErrorPage403 from './components/ErrorPage403';
+import { initWorkspaceRefreshToken } from './Workspace/WorkspaceRefreshToken';
+
+if (isEnabled('workspaceTokenServiceRefreshTokenAtLogin')) {
```

Comment:
other index pages (which are being used by other portal bundles) also needs this change?

Reply:
not sure. Is this one not used by all?

Reply:
no, some Data Commons have their own landing page, so they have their own index page (`covid19Index.jsx`, `nctIndex.jsx`, etc)

Reply:
so do we need to add this change there as well?

Reply:
The logic needs to happen for every Commons, not just the ones that use the default landing page

Reply:
Ok, I'll add that

Reply:
Or not... This change has been reverted in favor of the one in `src/Login/ProtectedContent.jsx` 😅